ssl/statem/extensions_clnt.c

```diff
@@ -538,7 +544,7 @@ static int add_key_share(SSL *s, WPACKET *pkt, unsigned int curve_id)
         assert(s->hello_retry_request);
         if (!s->hello_retry_request) {
             SSLerr(SSL_F_ADD_KEY_SHARE, ERR_R_INTERNAL_ERROR);
-            return 0;
+            return EXT_RETURN_FAIL;
```

Comment:
I bet I was confused by the git diff output that shows the prototype of the function containing the change, but it shows the *previous* prototype, and I happened to spot-check the one that was actually a valid complaint.  Or maybe the "various" just meant "multiple times within add_key_share()".
Regardless, I agree that add_key_share() is the only one that needs to change; thanks for putting together #3488.